app/src/main/res/drawable/ic_menu_info.xml

```diff
@@ -0,0 +1,12 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="20dp"
+    android:height="20dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillColor="#ffffff"
+        android:pathData="M12,2C6.48,2 2,6.48 2,12s4.48,10 10,10 10,-4.48 10,-10S17.52,2 12,2zM13,17h-2v-6h2v6zM13,9h-2L11,7h2v2z" />
```

Comment:
This looks like a duplicate of ic_info.xml (just with different size and color). Is it possible to re-use the resource?

Reply:
afaict, this is non-trivial:
- We could change it programmatically, but it's non-trivial to modify the Toolbar programmatically from where we'd need to access it in the code
- There is an app bar action menu item iconTint property, but it requires a too high API level (and doesn't solve the size problem)